apps/www/components/Article/Page.js

```diff
@@ -708,9 +708,8 @@ const ArticlePage = ({
           const format = meta.format
 
           const isFreeNewsletter = !!newsletterMeta && newsletterMeta.free
-          const showNewsletterSignupTop = isFreeNewsletter && !me && isFormat
-          const showNewsletterSignupBottom =
-            isFreeNewsletter && !showNewsletterSignupTop
```

Comment:
Can't understand this change, what the motiviation nor the thoughts behind these changes are. As me is no longer checked, I'm not sure if this still works as expected or not.

Reply:
`me` was checked to only show the signup box if you're logged in or not logged in. But it makes more sense to always show the newsletter-signup on a format page for a free newsletter, independent of login state

Reply:
now the logic should be: show the signup on top on the format page, show the signup box on the bottom on the article page, both only for free newsletters